lib/options.js

```diff
@@ -116,6 +116,15 @@ module.exports = {
         external: false,
         usage: ['CONVERSION']
       },
+      {
+        name: 'Include Auth info in Example request',
```

Comment:
We should use consistent case here - `Include auth info in example requests`

The description can be `Select whether to include authentication parameters in the example request`